tests/test_meta_tensor.py

```diff
@@ -488,6 +491,15 @@ def test_construct_with_pre_applied_transforms(self):
         m = MetaTensor(im, applied_operations=data[PostFix.transforms(key)])
         self.assertEqual(len(m.applied_operations), len(tr.transforms))
 
+    @SkipIfBeforePyTorchVersion((1, 8, 1))
+    def test_multiprocessing(self):
```

Comment:
Is this test alone enough to satisfy ourselves that multi-processing is supported?

Reply:
this is mainly to fix the current integration error, I think there might be more issues when we refactor more components, so I think maybe it's fine to merge this to `feature/MetaTensor` first...